README.md

```diff
@@ -1,8 +1,21 @@
 # Stedi EDI Bootstrap
 
-This repo contains an end-to-end configuration for building a full X12 EDI system using Stedi products. This implementation demonstrates one possible way to interact with Stedi's Products and APIs to achieve a typical EDI workload; your implementation may include some or all of these products depending on your particular systems and requirements.
+This repository contains an end-to-end configuration for building a full X12 EDI system using Stedi products. This implementation demonstrates one way to build an integration for the common read and write EDI use case. Your solution may differ, depending on your systems and requirements. We encourage you to [contact us](mailto:support@stedi.com) as you get started with Stedi, so we can help you design a custom integration.
 
-# Prerequisites & Deployment
+
+## Read and Write Workflow 
+
+The bootstrap workflow uses the following process to handle incoming EDI:
+
+1. An example trading partner uses [Stedi SFTP](https://www.stedi.com/products/sftp) to send EDI documents to a [Stedi bucket](https://www.stedi.com/products/buckets).
+2. New documents automatically invoke a [Stedi Function](https://www.stedi.com/products/functions) that contains custom code for the workflow. 
+3. The function calls [Stedi EDI Translate](https://www.stedi.com/products/edi-translate) to transform the EDI data into JSON. This process uses a [Stedi Guide](https://www.stedi.com/products/guides) to map EDI fields to JSON fields. 
+4. The function sends the JSON document to an internal webhook that you configure.
+
+The bootstrap uses a similar process in reverse to translate JSON documents to EDI and publish them to a bucket.
```

Comment:
nit: it might be good to specifically mention "handle outgoing EDI" here, to align with the description for incoming EDI above.

Reply:
excellent call - let me adjust accordingly!